docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,793 @@
+        "packageName": "coastal",
+        "version": "1.0.0-alpha1"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.0.0",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.0.0",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.0.0"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.1.0",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.1.0",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.1.0"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.0.1",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.0.1",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.0.1"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.channel",
+  "name": "stable",
+  "package": "coastal",
+  "entries": [
+    {
+      "name": "coastal.v1.0.1"
+    },
+    {
+      "name": "coastal.v1.1.0"
+    },
+    {
+      "name": "coastal.v1.0.0"
+    },
+    {
+      "name": "coastal.v1.0.0-alpha1"
+    }
+  ]
+}
+EOF
+```
+
+- Build and push the FBC image
+```sh
+docker build -t quay.io/operator-framework/coastal-catalog:latest -f catalog.Dockerfile . && \
+docker push quay.io/operator-framework/coastal-catalog:latest
+```
+
+Once again, we should see the `Catalog` update it's resolved reference and this time we expect that the `Operator` resource is automatically upgraded to the new `v1.0.1` bundle we added.
+
+## Change version range to pin installs/upgrades to `v1.1.z` releases
+Making changes to the `Operator` resource should result in a re-reconciliation of the resource
+which should result in another resolution loop. To see this, lets update the version range
+on our `Operator` resource to `1.1.x` with:
+```sh
+kubectl apply -f - <<EOF
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: coastal
+spec:
+  packageName: coastal
+  version: 1.1.x
+EOF
+```
+
+After this change has been applied, we should eventually see the `Operator` resource is updated
+and says that we have installed the `v1.1.0` bundle we created and added to the catalog earlier.
+
+You can watch this change happen with:
+```sh
+kubectl get operator/coastal -o yaml -w
+```
+
+## Attempting a major version upgrade by changing the version range
+When an `Operator`'s `spec.upgradeConstraintPolicy` is set to `Enforce`, operator-controller
```

Comment:
Downstream, we have been using "Operator Controller".

Reply:
I think this is addressed in [08e8dbb](https://github.com/operator-framework/operator-controller/pull/533/commits/08e8dbba77679d610d04f2b8002613fe94421aff)